<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hoi4 TagSwitch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="FOWBox">
       <property name="text">
        <string>FOW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="AllowTraitsBox">
       <property name="text">
        <string>Allow Traits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="DebugBox">
       <property name="text">
        <string>Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OpenTutorialButton">
       <property name="text">
        <string>Open Tutorial</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="TagBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="TagButton">
       <property name="text">
        <string>Tag Switch</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect HOI4</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnectedLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hoi4 -</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>OpenTutorialButton</tabstop>
  <tabstop>TagButton</tabstop>
  <tabstop>TagBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
